easybuild/easyconfigs/p/Python/Python-3.6.2-GCCcore-6.4.0-bare.eb

```diff
@@ -0,0 +1,56 @@
+name = 'Python'
+version = '3.6.2'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+
+description = """
+ Python is a programming language that lets you work more quickly and
+ integrate your systems more effectively.
+
+Note: This module is meant to provide a builddependency for other Python
+      modules while using EasyBuild's --minimaltoolchain option.  Modules
+      built with it will require the full Python later
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['7919489310a5f17f7acbab64d731e46dca0702874840dadce8bd4b2b3b8e7a82']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.19.3'),
+    ('zlib', '1.2.11'),
+]
+
+osdependencies = [
+    # rely upon distribution for timely security updates
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+# We hide this by default since users should not use it in production,
+# high-performance Python should be delivered at compiler level with
+# a default extension set
+# hidden = True
+
+# bare installation: only known module deps for GCCcore tools included
+exts_list = []
+
+# Until such time that EasyBuild accepts 'rpath' as a toolchainopt...
+# add RPATH so this can be used as a builddependency for "bare" PythonPackages
```

Comment:
Given that EB doesn't update LD_LIBRARY_PATH with builddependencies, then unless the RPATH is set in the python binary, any PythonPackages built on top of it will fail during the sanity check (python -c 'import foo') due to not finding libpythonX.Y.so.    I've have 100+ EBs dealing with Python-bare.  The EBs in 3.3.1 barely scratch the surface, probably due to this fact.  I'm telling you, from experience, if you want to build with --minimal-toolchains then you need the RPATH in python-bare.

I don't care how the RPATH is added, but it needs to be added given the current framework for --minimal-toolchains to be useful.